backend/src/services/stripe/stripe.service.ts

```diff
@@ -0,0 +1,72 @@
+import StripeApi from 'stripe';
+
+import {
+  PaymentCurrency,
+  PaymentMethod,
+  PaymentMode,
+  PaymentUnit,
+} from '~/common/enums/enums';
+
+const QUANTITY_OF_ITEMS = 1;
+const OPERATION_NAME = 'Replenish your funds on GuruHub';
+
+type Constructor = {
+  secretKey: string;
+  successUrl: string;
+  cancelUrl: string;
+  apiVersion: '2022-08-01';
```

Comment:
```suggestion
  apiVersion: string;
```
:thinking:

Reply:
That Api requires this type 🤑

Reply:
Yes, but you're passing the type from ENV, I think there's no need to specify it here